src/main/java/uk/gov/hmcts/reform/sscs/model/CaseCategoryType.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+@Getter
+public enum CaseCategoryType {
+
+    CASETYPE("caseType"),
+    CASESUBTYPE("caseSubType");
+
+    private final String catagoryLabel;
+
+    CaseCategoryType(String catagoryLabel) {
+        this.catagoryLabel = catagoryLabel;
+    }
+
+    public static CaseCategoryType getByLabel(String label) {
```

Comment:
Is this used anywhere?

You can use equalsIgnoringCase to filter.

Is it safe to return null here?